crew

```diff
@@ -843,8 +846,8 @@ def resolve_dependencies_and_build
   ensure
     #cleanup
     unless @opt_keep
-      FileUtils.rm_rf Dir.glob("#{CREW_BREW_DIR}/*")
-      FileUtils.mkdir_p "#{CREW_BREW_DIR}/dest" #this is a little ugly, feel free to find a better way
+      FileUtils.rm_rf "#{Dir.glob('#{CREW_BREW_DIR}/*')}#{@fileutils_verbose}"
+      FileUtils.mkdir_p '#{CREW_BREW_DIR}/dest'#{@fileutils_verbose}" #this is a little ugly, feel free to find a better way
```

Comment:
Variables should always be enclosed by double quotes.  The single quotes make it literal.